### ISSUE TITLE
[Bug] Wrong font in chat list and composer

### ISSUE THREAD

**Issue by tprudentova** [linagora/twake-on-matrix#2159]
2.6.10, chat.stg.lin-saas.com, all platforms 

The font in chat list and the composer currently is regular 

Expected: the font is medium 

![5346139127445316278](https://github.com/user-attachments/assets/bc9170ad-fea9-4cbb-a48a-2df4fba7e09c)

**Comment by hoangdat** [comment]
what is your composer @tprudentova ?

**Comment by Bobpodvalnyi** [comment]
and the text in composer while you typing has to be the same "Medium"but not "Regular"  
![image](https://github.com/user-attachments/assets/b4383a96-cf34-4daf-9046-2f5f44293be0)

**Comment by Bobpodvalnyi** [comment]
and the text in composer while you typing has to be the same "Medium"but not "Regular"
![image](https://github.com/user-attachments/assets/0e3dacb0-8ccc-40d6-84b7-b3fb942f37c7)

**Comment by hoangdat** [comment]
### Figma: `M3/body/large`
<img width="587" alt="Screenshot 2024-11-27 at 11 05 20" src="https://github.com/user-attachments/assets/709b91af-3614-419f-aace-3cb6e9b50bc8">

### In Flutter: `M3/body/large`
![image](https://github.com/user-attachments/assets/fce7cb86-a862-4bd0-b7dc-a9fbe2236bfa)

In coding, to strictly follow M3 and avoiding boilerplate, we always depend on the name of typhography: `M3/body/large`. The same typography but the different value.

The problem here: Figma have updated version, but Flutter engine (which use for TWAKE CHAT) still not. I also discussed this with Khanh before: `Please dont update Design System frequently`.

**Comment by Bobpodvalnyi** [comment]
![Screenshot 2024-11-27 at 15 28 40](https://github.com/user-attachments/assets/2f14ed7e-fae3-4e64-87ec-60d3e43b1ede)
could you explain me Why do you have "inter_regular" listed but not inter in the family.
Inter regular doesn't have a weight of 500

**Comment by Bobpodvalnyi** [comment]
There have been no updates to the system design file for the last 11 months. So it's not about updates @hoangdat

**Comment by Bobpodvalnyi** [comment]
@hoangdat  By your logic, the two texts should not be different since they both have a weight of 500
But the text in the dough message really has a weight of 500 but in the input field and in the chat list you have gone something wrong.
![Screenshot 2024-11-27 at 16 37 27](https://github.com/user-attachments/assets/8a17ddec-c4ee-40ee-a140-633b44f08823)

**Comment by DianaPotokina** [comment]
yes, fonts are different. @hoangdat find the real reason why it's like this. It needs to be fixed
